quest_test/test_persistence.py

```diff
@@ -1,14 +1,31 @@
 import asyncio
 from pathlib import Path
+from typing import Callable
+from dotenv import load_dotenv
```

Comment:
`dotenv` is better for the end-user, not a library. The library should assume all the necessary variables are already in the environment.

Reply:
I was using this for testing, how should we run the tests so AWS works correctly? I looked at moto